make/CreateJmods.gmk

```diff
@@ -226,6 +226,12 @@ else
   JMOD_FLAGS += --exclude '**{_the.*,_*.marker*,*.diz,*.debuginfo,*.dSYM/**,*.dSYM}'
 endif
 
+# Unless we are creating a very large module, use the small tool JVM options
+JMOD_SMALL_FLAGS=
```

Comment:
Missed this at first, but we try to stick to := assignment unless we really need the lazy evaluation. Not really an issue here other than consistency.

Reply:
Done in new commit.